Tests/aweXpect.Reflection.Tests/ThatMethod.IsSealed.Tests.cs

```diff
@@ -0,0 +1,56 @@
+using System.Reflection;
+using aweXpect.Reflection.Tests.TestHelpers.Types;
+
+namespace aweXpect.Reflection.Tests;
+
+public sealed partial class ThatMethod
+{
+	public sealed class IsSealed
+	{
+		public sealed class Tests
```

Comment:
The tests for the negated case using `DoesNotComplyWith` (see [ThatFields.ArePublic.NegatedTests](https://github.com/aweXpect/aweXpect.Reflection/blob/7ca54066e02e76194e0e78a2d1dc751c13b8009e/Tests/aweXpect.Reflection.Tests/ThatFields.ArePublic.Tests.cs#L45) for an example) are missing.